src/routes/index.svelte

```diff
@@ -129,8 +109,12 @@
         goto(queryParams)
     }
 
-    const getNewSymbols = (stocks, history) => {
-        return stocks.filter(stock => !history.get(stock))
+    const isValidSymbol = symbol => symbol.length > 0 && symbol.length <= 5
```

Comment:
```suggestion
    const isValidSymbol = (symbol) => symbol.length > 0 && symbol.length <= 5
```

Reply:
Linting will chooses no brackets

Reply:
Thought so, but is there a reason it then allows for the following earlier in the file: 
`const convertMsToS = (n) => n / 1000`